docs/external-node.md

```diff
@@ -0,0 +1,532 @@
+  - [Bypass Antrea NetworkPolicy](#bypass-antrea-networkpolicy)
+- [RBAC for antrea-agent](#rbac-for-antrea-agent)
+- [VM network configuration](#vm-network-configuration)
+- [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [Install Antrea Agent on VM](#install-antrea-agent-on-vm)
+  - [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster)
+  - [Installation on Linux VM](#installation-on-linux-vm)
+    - [Prerequisites](#prerequisites-1)
+    - [Installation](#installation)
+  - [Installation on Windows VM](#installation-on-windows-vm)
+    - [Prerequisites](#prerequisites-2)
+    - [Installation](#installation-1)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+# What is ExternalNode?
+
+`ExternalNode` is a CRD API that enables Antrea to manage the network connectivity
+and security on a Non Kubernetes Node (like a virtual machine or a bare-metal
+server). It supports specifying which network interfaces on the external Node
+are expected to be guarded with Antrea NetworkPolicy rules. The virtual machine
+or bare-metal server represented by an `ExternalNode` resource can be either
+Linux or Windows. "external Node" will be used to represent such a virtual
+machine or a bare-metal server in this document.
+
+Antrea NetworkPolicy is applied to an external Node via the `ExternalEntity`
+resource. `antrea-controller` creates an `ExternalEntity` resource for each
+network interface defined in the `ExternalNode` resource.
+
+`antrea-agent` is running on the external Node, and it controls network
+connectivity and security by attaching the network interface(s) to an OVS bridge.
+A new OpenFlow pipeline has been implemented, dedicated to the ExternalNode
+feature.
+
+You may be interested in using this capability for the below scenarios:
+
+- To apply Antrea NetworkPolicy to an external Node.
+- You want the same security configurations on the external Node for all
+  Operating Systems.
+
+This guide demonstrates how to configure `ExternalNode` to achieve the above
+result.
+
+# Prerequisites
+
+`ExternalNode` is introduced in v1.8 as an alpha feature. A feature gate
+`ExternalNode` must be enabled on the `antrea-controller` and `antrea-agent`in
+the configuration. The configuration for `antrea-controller` is modified in the
+`antrea-config` ConfigMap as follows for the feature to work:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: antrea-config-dcfb6k2hkm
+  namespace: kube-system
+data:
+  antrea-controller.conf: |
+    featureGates:
+      ExternalNode: true
+```
+
+Service `antrea` is created for `antrea-controller` to accept connections from
+`antrea-agent`, and the type is `ClusterIP` by default. Since an external Node
+is running outside the Kubernetes cluster, the `ClusterIP` address is not
+accessible. So a `NodePort` or `LoadBalancer` typed Service for `antrea` is
+required.
```

Comment:
for this paragraph:

> The `antrea-controller` implements the `antrea` Service, which accepts connections from each `antrea-agent` and is an important part of the NetworkPolicy implementation. By default, the `antrea` Service has type `ClusterIP`. Because external Nodes run outside of the Kubernetes cluster, they cannot directly access the `ClusterIP` address. Therefore, the `antrea` Service needs to become externally-accessible, by changing its type to `NodePort` or `LoadBalancer`.

Reply:
update as suggested.